src/Nest/Ingest/Processors/KeyValueProcessor.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using Newtonsoft.Json;
+
+namespace Nest
+{
+	[JsonObject(MemberSerialization.OptIn)]
+	[JsonConverter(typeof(ProcessorJsonConverter<KeyValueProcessor>))]
+	public interface IKeyValueProcessor : IProcessor
```

Comment:
Should this be called `KVProcessor` to [align with the docs?](https://www.elastic.co/guide/en/elasticsearch/reference/5.x/kv-processor.html)